tools/pipelines/templates/include-publish-docker-service-steps.yml

```diff
@@ -53,13 +53,13 @@ jobs:
         - task: Docker@1
           displayName: Tag image as '$(containerTagSuffix)'
           inputs:
-            command: 'tag'
+            command: 'Tag image'
             arguments: '${{ parameters.containerTag }} ${{ parameters.containerRegistryUrl }}/${{ parameters.containerName }}:$(containerTagSuffix)'
 
         - task: Docker@1
           displayName: Tag image as 'latest'
           inputs:
-            command: 'tag'
+            command: 'Tag image'
```

Comment:
Looked at the docs because I couldn't believe this was right... still incredulous at the schema for that field. 😳